account_invoice_transmit_method/views/account_move.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright 2017-2020 Akretion France (http://www.akretion.com/)
+  @author: Alexis de Lattre <alexis.delattre@akretion.com>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<odoo>
+
+
+<record id="view_move_form" model="ir.ui.view">
+    <field name="name">Add Transmit Method on Invoice form view</field>
+    <field name="model">account.move</field>
+    <field name="inherit_id" ref="account.view_move_form"/>
+    <field name="arch" type="xml">
+        <field name="journal_id" position="after">
+            <field name="transmit_method_id" widget="selection"
+                domain="[('customer_ok', '=', True)]"
```

Comment:
While testing I noticed a problem with this domain, it is never set properly. And will always be set to the next one below `domain="[('supplier_ok', '=', True)]`. So a transmit_method declared has customer only will not be displayed.
It seems that Odoo does not handle properly the same field declared twice on the same view with different domains. I didn't dig into the source code to know why.

Reply:
Indeed Odoo doesn't support that, fields are unique in a view, and Odoo will consider only the last one.
If the goal is to have a different domain depending of the invoice type, we could overload the `fields_view_get` method?

Reply:
This field is defined twice so it is placed in a different position on customer vs vendor invoice.
This was not the case in previous version.
But is it really necessary ? Maybe the idea behind it is that it is more important to see the transmit method for customer invoices because it will require an action from the user to transmit the invoice ? 
In previous version it was 
I tend to think that if it is present for both type of invoices in the `Other info` tab

Reply:
@alexis-via Thank you for the fix. I actually did not know you could do this in an `onchange`.
Could you take care of the linting errors in the xml files, I think it is the only thing that prevents this PR to be merged